packages/platforms/accton/x86-64/as5812-54t/onlp/builds/x86_64_accton_as5812_54t/module/src/make.mk

```diff
@@ -1,9 +1,10 @@
 ###############################################################################
 #
-# 
+#
 #
 ###############################################################################
 
 LIBRARY := x86_64_accton_as5812_54t
 $(LIBRARY)_SUBDIR := $(dir $(lastword $(MAKEFILE_LIST)))
+GCC_NO_WERROR := 1
```

Comment:
To ignore some warning such as format-truncation, same for other `GCC_NO_WERROR`
We can also fix the code, but need some time